src/main/java/net/conjur/api/clients/ResourceClient.java

```diff
@@ -20,28 +21,39 @@
 public class ResourceClient implements ResourceProvider {
 
     private WebTarget secrets;
-    private final Endpoints endpoints;
+    private Endpoints endpoints;
 
-    public ResourceClient(final Credentials credentials, final Endpoints endpoints) {
+    public ResourceClient(Credentials credentials, Endpoints endpoints) {
+        this(credentials, endpoints, null);
+    }
+
+    public ResourceClient(Credentials credentials, Endpoints endpoints, SSLContext sslContext) {
         this.endpoints = endpoints;
 
-        init(credentials);
+        init(credentials, sslContext);
+    }
+
+    // Build ResourceClient using a Conjur auth token
+    public ResourceClient(Token token, Endpoints endpoints) {
+        this(token, endpoints, null);
     }
 
     // Build ResourceClient using a Conjur auth token
-    public ResourceClient(final Token token, final Endpoints endpoints) {
+    public ResourceClient(Token token, Endpoints endpoints, SSLContext sslContext) {
```

Comment:
Same question - why the removal of `final`?

Reply:
Mistake, fixed.